### PR TITLE
Use belt when possible

### PR DESCRIPTION
**Pull request by j-m-hoffmann**

### CODE REVIEW

File: bs/__tests__/tablecloth_test.ml [hunk 1]
```diff
@@ -17,7 +17,9 @@ let () =
     test "indexedMap two elements" (fun () -> expect (List.indexedMap ~f:(fun i _ -> i) ['a';'b']) |> toEqual [0;1]);
 
     test "init empty list" (fun () -> expect (List.init []) |> toEqual None);
+    (* This seems kinda wrong, if this is correct, delete the comments, proposed solution in tablecloth.ml *)
     test "init one element" (fun () -> expect (List.init ['a']) |> toEqual (Some []));
```
Comment (pbiggar) [inline bs/__tests__/tablecloth_test.ml]: `List.init` is like `List.tail`, so whatever List.tail does, List.init should do too.

`List.tail []` => `None`
`List.tail [1]` => []`
`List.tail [1;2]` => [2]`

`List.init []` => `None`
`List.init [1]` => []`
`List.init [1;2]` => [1]`

File: bs/src/tablecloth.ml [hunk 2]
```diff
@@ -276,8 +276,7 @@ module List = struct
 
   let insert_at = insertAt
 
-  let splitWhen ~(f : 'a -> bool) (l : 'a list) : ('a list * 'a list) option
-      =
+  let splitWhen ~(f : 'a -> bool) (l : 'a list) : ('a list * 'a list) option =
```
Comment (pbiggar) [inline bs/src/tablecloth.ml]: For the styling, I'm using ocamlformat with these settings: https://github.com/darklang/tablecloth/issues/7

We should standardize on them at some point, but until then any format is fine.

File: bs/__tests__/tablecloth_test.ml (hunk 1 above)
Comment (j-m-hoffmann) [reply bs/__tests__/tablecloth_test.ml]: thats what i thought so there was a bug

Comment (j-m-hoffmann) [reply bs/__tests__/tablecloth_test.ml]: ah no, everything worked as intended

**Pull request merged**